utils/move_and_process.py

```diff
@@ -256,10 +256,11 @@ def create_videofile(video_id, data):
     rq('POST', '/videofiles/', data=data)
 
 def update_videofile(data):
+    videofile_id = data['id']
     if args.no_api:
-        logging.debug("NO API - creating videofile %s -> %r" % (video_id, data))
+        logging.debug("NO API - updating videofile %s -> %r" % (videofile_id, data))
         return
-    rq('PATCH', '/videofiles/', data=data)
+    rq('PATCH', '/videofiles/%s' % data['id'], data=data)
```

Comment:
I'd reuse the `videofile_id` you created above.

Reply:
Good idea.